src/julia-syntax.scm

```diff
@@ -1609,6 +1609,16 @@
        (error (string "invalid assignment location \"" (deparse lhs) "\"")))
       (else
        (case (car lhs)
+         ((globalref)
+          ;; M.b =
+          (let* ((a   (cadr lhs))
+                 (b   (caddr lhs))
+                 (rhs (caddr e))
+                 (rr  (if (or (symbol-like? rhs) (atom? rhs)) rhs (make-ssavalue))))
+            `(block
+              ,.(if (eq? rr rhs) '() `((= ,rr ,(expand-forms rhs))))
+              (call (core setfield!) ,a (quote ,b) ,rr)
```

Comment:
What commit are you talking about? https://github.com/JuliaLang/julia/issues/10403 is an issue not a PR and all commits referenced in that issue are older than this one and I'm pretty sure `$(GlobalRef(...)) = ...` is not handled anywhere when I wrote this.